src/test/java/net/snowflake/client/jdbc/CustomProxyLatestIT.java

```diff
@@ -452,13 +452,25 @@ public void testProxyConnectionWithoutProxyPortOrHost()
             + "&proxyUser=testuser1&proxyPassword=test"
             + "&useProxy=true";
     try {
-      runAzureProxyConnection(connectionUrl, false);
+      runAzureProxyConnection(connectionUrl, false, true);
       fail();
     } catch (SQLException e) {
       assertEquals(SqlState.CONNECTION_EXCEPTION, e.getSQLState());
     }
   }
 
+  @Test
+  @Ignore
+  public void testProxyConnectionWithJVMParameters() throws SQLException, ClassNotFoundException {
+    String connectionUrl =
+        "jdbc:snowflake://aztestaccount.east-us-2.azure.snowflakecomputing.com/?tracing=ALL";
+    System.setProperty("http.useProxy", "true");
+    System.setProperty("http.proxyHost", "localhost");
+    System.setProperty("http.proxyPort", "8080");
+    System.setProperty("http.nonProxyHosts", "*.snowflake.com");
+    runAzureProxyConnection(connectionUrl, false, false);
```

Comment:
It is confusing that the name of the test is `testProxyConnectionWithJVMParameters` and here `usesJVMProperties = false`. Can you explain?

Reply:
Yes- I've now changed this variable to usesIncorrectJvmProperties. Within the function runAzureProxyConnection(), I set incorrect JVM properties to prove that the connection properties override the JVM properties. So in this case, I set the correct properties before calling runAzureProxyConnection() and I don't want those incorrect properties to get set internally.